charts/kruise/v0.6.1/templates/apps.kruise.io_uniteddeployments.yaml

```diff
@@ -752,6 +752,16 @@ spec:
                   required:
                   - spec
                   type: object
+                deploymentTemplate:
```

Comment:
Let us not change this. The ud support of deployment will go live in next release.